src/sentry/tasks/post_process.py

```diff
@@ -195,11 +195,7 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
 
             for plugin in plugins.for_project(event.project):
                 plugin_post_process_group(
```

Comment:
Are you sure this behavior is backward compatible? is_sample was passed to the plugin. Are we sure it was never used by anyone ?

Reply:
I believe it's only expected by the Jira plugin. This change needs to go first https://github.com/getsentry/sentry-plugins/pull/513